k8s/configs/prometheus/values.yaml

```diff
@@ -120,8 +120,8 @@ grafana:
         revision: 7
       Kong:
         datasource: Prometheus
-        gnetId: 7424
-        revision: 9
+        gnetId: 17269
```

Comment:
Hey, quick question - who owns this dashboard? 
Glif or Kong?

Reply:
I prefer to reuse Kong ones with needed revision instead of creating new one.
Or plz give a reason to have our own version :thinking: